frontend/src/components/dashboard/DashboardBlock.tsx

```diff
@@ -8,66 +8,95 @@ import LongCardCell from "../utils/LongCardCell";
 import Term from "../utils/Term";
 import Link from "../utils/Link";
 
+import { Translate } from "react-localize-redux";
+
 export interface Props {
   className?: string;
 }
 
 const DashboardBlock = ({ className }: Props) => (
-  <DatabaseConsumer>
-    {(context) => (
-      <DashboardCard
-        className={`block-card ${className || ""}`}
-        iconPath="/static/images/icon-blocks.svg"
-        title="Blocks"
-        headerRight={
-          <Link href="/blocks">
-            <a>View All</a>
-          </Link>
-        }
-      >
-        <Row noGutters>
-          <Col xs="6" md="12">
-            <LongCardCell
-              title={
-                <Term
-                  title={"Block Height"}
-                  text={
-                    <>
-                      <p>{`The most recent block height recorded to the blockchain.`}</p>
-                      <p>{`The block height is a sequential number of the most recent block in the blockchain.`}</p>
-                      <p>{`For example, a block height of 1000 indicates that up to 1001 blocks may exist in the blockchain (genesis + blocks 0-1000).
-                        In NEAR, there is not guaranteed to be a block for each sequential number, e.g. block 982 does not necessarily exist.`}</p>
-                    </>
+  <Translate>
+    {({ translate }) => (
+      <DatabaseConsumer>
+        {(context) => (
+          <DashboardCard
+            className={`block-card ${className || ""}`}
+            iconPath="/static/images/icon-blocks.svg"
+            title={translate("model.blocks.title").toString()}
+            headerRight={
+              <Link href="/blocks">
+                <a>
+                  <Translate id="button.view_all" />
+                </a>
+              </Link>
+            }
+          >
+            <Row noGutters>
+              <Col xs="6" md="12">
+                <LongCardCell
+                  title={
+                    <Term
+                      title={translate(
+                        "component.dashboard.DashboardBlock.block_height.title"
+                      ).toString()}
+                      text={
+                        <>
+                          <p>
+                            {translate(
+                              "component.dashboard.DashboardBlock.block_height.desc1"
+                            )}
+                          </p>
+                          <p>
+                            {translate(
+                              "component.dashboard.DashboardBlock.block_height.desc2"
+                            )}
+                          </p>
+                          <p>
+                            {translate(
+                              "component.dashboard.DashboardBlock.block_height.desc3"
+                            )}
+                          </p>
```

Comment:
In `frontend/src/components/accounts/AccountDetails.tsx` you [used `{ renderInnerHtml: true }`](https://github.com/near/near-explorer/pull/679/files?diff=split&w=1#diff-2251c6f194c4bc059c717310df2d85119d9d52070a1004c36ff83519cda22431R138-R142). I feel that is a better option to use since all these `desc*` are part of the same documentation text

Reply:
I remember we have tried put the html string into JSON file, but failed to render correctly. It's probably caused by how `<Term />` component is implemented, and I can improve this today.

Reply:
After testing, we figure out the previous failure was caused by one bug in the fallback mechanism of `react-localize-redux`. (The fallback will not work for html strings.) It's working now if we add html translation correctly for both the default language and active language